R/correlations.R

```diff
@@ -21,7 +21,7 @@
 correlation <- function(scores,
                         metrics = NULL,
                         digits = NULL) {
-  metrics <- get_metrics(scores)
+  metrics <- get_score_names(scores)
```

Comment:
why is metrics an argument if it immediately gets overwritten? Shouldn't this (i.e. get_score_names be the default arg for metrics instead).

Reply:
also this arg needs renaming (already an issue to address this?) but not as part of this PR.

Reply:
Excellent point. It might be mentioned somewhere, but I'll make it a separate issue (#589) so it doesn't get buried somewhere.